### PR TITLE
Add unit tests for RabbitMQStore class

### PR DESCRIPTION
**Pull request by GDLMadushanka**
## Automation tests
 - Unit tests 
   > 24%

### CODE REVIEW

File: modules/core/src/test/java/org/apache/synapse/message/store/RabbitMQStoreTest.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        Assert.assertEquals("Comparing MessageConsumer id", "[null-C-1]", messageConsumer.getId());
+    }
+
+    /**
+     * Create axis2MessageContext
+     */
+    @Test
+    public void testNewAxis2Mc() {
+        MessageContext messageContext = rabbitMQStore.newAxis2Mc();
+        Assert.assertEquals(messageContext.getConfigurationContext(), configurationContext);
```
Comment (chanikag) [inline modules/core/src/test/java/org/apache/synapse/message/store/RabbitMQStoreTest.java]: Add an Assert message here?

File: modules/core/src/test/java/org/apache/synapse/message/store/RabbitMQStoreTest.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    }
+
+    @Test
+    public void testRemove() {
+        Assert.assertNull(rabbitMQStore.remove());
```
Comment (chanikag) [inline modules/core/src/test/java/org/apache/synapse/message/store/RabbitMQStoreTest.java]: Add an Assert message here?

File: modules/core/src/test/java/org/apache/synapse/message/store/RabbitMQStoreTest.java [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+     */
+    @Test
+    public void testNewAxis2Mc() {
+        MessageContext messageContext = rabbitMQStore.newAxis2Mc();
+        Assert.assertEquals("MessageContext should be initiated with configurationContext",messageContext
+                .getConfigurationContext(), configurationContext);
+    }
+
+    //Following test cases assert currently not implemented methods which are just returning null
+    @Test
```
Comment (pamod) [inline modules/core/src/test/java/org/apache/synapse/message/store/RabbitMQStoreTest.java]: Describe the following test cases

Comment (GDLMadushanka) [reply modules/core/src/test/java/org/apache/synapse/message/store/RabbitMQStoreTest.java]: Methods tested in these tests are not implemented yet. I add these test case for coverage purposes and for the ease of changing them when methods are actually implemented.

**Pull request merged**